tg_global_gsql.yaml

```diff
@@ -0,0 +1,21 @@
+---
```

Comment:
Why is this file named .yaml, while the other files are all .yml? Shall we make them consistent? cc @wyatt-joyner-tg @LinYu01

Reply:
Ah, thank you for catching this.